doc/topics/core-structure.md

```diff
@@ -0,0 +1,202 @@
+# Node Core's Internal Structure
+
+This document covers the basics of how Node.js is structured, in an attempt to
```

Comment:
Nit: 

* remove the comma
* consider removing everything after the comma too
* s/covers the basics/explains/